analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ExecutingSqlQueries_Net46.vb

```diff
@@ -244,34 +244,37 @@ Namespace Tests.Diagnostics
         End Sub
 
         Public Sub ConcatAndStringFormat(ByVal connection As SqlConnection, ByVal param As String)
-            Dim sensitiveQuery As String = String.Format("INSERT INTO Users (name) VALUES (""{0}"")", param)
-            Dim command = New SqlCommand(sensitiveQuery)                                                        ' Noncompliant
-            command.CommandText = sensitiveQuery                                                                ' Noncompliant
+            Dim sensitiveQuery As String = String.Format("INSERT INTO Users (name) VALUES (""{0}"")", param)    ' Secondary [1,2,3,4] {{SQL Query is dynamically formatted and assigned to sensitiveQuery.}}
+            '   ^^^^^^^^^^^^^^
+            Dim command = New SqlCommand(sensitiveQuery)                                                        ' Noncompliant [1]
+            command.CommandText = sensitiveQuery                                                                ' Noncompliant [2]
 
-            Dim stillSensitive As String = sensitiveQuery
-            command.CommandText = stillSensitive                                                                ' Noncompliant
+            Dim stillSensitive As String = sensitiveQuery                                                       ' Secondary [3] {{SQL query is assigned to STILLsENSITIVE.}}
+            '   ^^^^^^^^^^^^^^
+            command.CommandText = STILLsENSITIVE                                                                ' Noncompliant [3] ^13#19
```

Comment:
This secondary message is confusing here. The query is assigned to `stillSensitive` and then read from or propagated through `STILLsENSITIVE`.